rosdep/python.yaml

```diff
@@ -2010,6 +2010,12 @@ python-joblib:
   fedora: [python-joblib]
   gentoo: [dev-python/joblib]
   ubuntu: [python-joblib]
+python-jsonpickle:
+  arch: [python-jsonpickle]
+  debian: [python-jsonpickle]
+  fedora: [python-jsonpickle]
+  gentoo: [dev_python/jsonpickle]
```

Comment:
This should be `dev-python`, not `dev_python`, no?

Reply:
Good catch! I am just going to edit it.

Reply:
@nuclearsandwich thx for fixing and @cottsay thx for the catch